importer.py

```diff
@@ -205,9 +215,9 @@ def save_artists_callback(importer, spotify_artists):
         self._add_items_to_spotify(artists, self.not_imported['Artists'], save_artists_callback)
 
     def import_all(self):
-        for item in self._importing_items.values():
-            item()
-
+        if self.yandex_client is not None:
```

Comment:
we should not call `import_all` if there is no YM client. the suggested if statement in the prev comment should help

Reply:
3c1d76b